src/components/MemberCard.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import SocialIcons from './SocialIcons.vue'
+
+defineProps<{
+  member: string
+}>()
+
+const socials = ['github', 'linkedin', 'twitter', 'website']
+
+const { t } = useI18n()
+</script>
+
+<template>
+  <div
+    class="overflow-hidden rounded-5px bg-#d8dee9 dark:bg-#3b4252 p-32px font-sans"
+    data-test="team-card"
+    :data-test-member-name="`team-member-${member}`"
+  >
+    <div class="relative z-0 flex h-auto items-center justify-center rounded-full mx-2em my-auto text-center">
+      <div
+        :data-test="`team-member-${member}-index-photo`"
+        class="block w-128px h-128px rounded-full bg-center bg-cover"
+        :style="`background-image: url( ${$t(`team.${member}.image`)} );`"
+      />
+    </div>
+    <div class="relative z-1 flex flex-col items-center justify-center gap-4px">
+      <h2
+        class="text-1.5rem font-700"
+        :data-test="`team-member-${member}-name`"
+      >
+        {{ $t(`team.${member}.name`) }}
+      </h2>
+
+      <SocialIcons :member="member" :socials="socials" />
+
+      <router-link
+        class="w-fit px-15px py-5px border-1px border-solid border-transparent rounded-full bg-#586379 text-#fff text-1rem font-700 hover:bg-#2e3440"
```

Comment:
No need to declare `text-#fff` this is simply `text-white` see [Text Color on Tailwind docs](https://tailwindcss.com/docs/text-color).

This rule is complicated: `hover:bg-#2e3440` what happens if one day we want to change this element into a component but we want to have different colours on hover? We would have to go to each component and modify it. Better to declare a global SCSS class that allows you to declare in the `<style scoped lang="scss">` and their respective classes to apply it.

Also, this colours have already been [declared on the `nord.scss` file](https://github.com/rennavittorio/schrodinger-hat-website/blob/17f21671b8415870af8f75e3521fd350e2e67a4f/src/styles/nord.scss#L38) which [is then imported globally] so you could easily add them on a specific SCSS class:

```vue
<style scoped lang="scss">
.my-class {
  &:hover {
    background-color: $dark-bg-primary;
  }
}
</style>
```

Also, does this component implement the change between light/dark theme properly?

Remember, refactoring to UnoCSS doesn't mean SCSS shouldn't be used at all!

Reply:
[chore(config): add brand color to unocss config](https://github.com/Schrodinger-Hat/schrodinger-hat-website/pull/94/commits/c165a857c2d0091962155a19818b15e23e8a1c89)
What do you think about this solution, and moving project variables into unocss config? 
Let's take full advantage of this cool tool! @Readpato